src/test/scala/com/microsoft/hyperspace/index/CreateIndexTests.scala

```diff
@@ -87,6 +85,27 @@ class CreateIndexTests extends SparkFunSuite with SparkInvolvedSuite {
     assert(exception.getMessage.contains("Index config is not applicable to dataframe schema"))
   }
 
+  test("Index creation passes with columns of different case if case-sensitivity is false.") {
+    hyperspace.createIndex(df, IndexConfig("index1", Seq("qUeRy"), Seq("ImpRS")))
+    val indexes = hyperspace.indexes.where(s"""name = "${indexConfig1.indexName}" """)
```

Comment:
nit: does the following work?
```
val indexes = hyperspace.indexes.where(s"name = '${indexConfig1.indexName}'")
```

Reply:
yes it works. Fixed here and other places.thanks @imback82 ,